mappings/net/minecraft/advancement/Advancement.mapping

```diff
@@ -28,7 +28,7 @@ CLASS net/minecraft/class_161 net/minecraft/advancement/Advancement
 	METHOD method_690 addChild (Lnet/minecraft/class_161;)V
 		ARG 1 child
 	METHOD method_691 getRewards ()Lnet/minecraft/class_170;
-	CLASS class_162 Task
+	CLASS class_162 Builder
```

Comment:
Yeah good rename, I named it a bad name previously.

This is not only a builder but a mutable representation of advancement before their parent relationships are set up, though calling it builder is good enough.